funnel/templates/label_badge.html.jinja2

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<head>
+  <script src='https://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js'></script>
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
+  <style>
+    @page {
+      size: 62mm 68mm;  /* width height */
```

Comment:
I think this should come from page config that's stored in the project. It's okay for now, but file a ticket for configurable badge layouts.